peachjam/views/article.py

```diff
@@ -79,6 +80,23 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class ArticleAttachmentDetailView(DetailView):
+    model = ArticleAttachment
+
+    def get_object(self, queryset=None):
+        return get_object_or_404(
+            ArticleAttachment,
+            document__date=self.kwargs["date"],
+            document__slug=self.kwargs["slug"],
+            document__author__username=self.kwargs["author"],
+            pk=self.kwargs["pk"],
+            filename=self.kwargs["filename"],
+        )
+
+    def render_to_response(self, context, **response_kwargs):
+        return FileResponse(self.object.file, filename=self.object.filename)
```

Comment:
can you add a long-lived caching header hear? eg 1 year, like our other static files?

Reply:
added